autoload/clap/provider/bcommits.vim

```diff
@@ -18,13 +18,7 @@ function! s:bcommits.source() abort
   endif
 
   let s:current = bufname(g:clap.start.bufnr)
-  let managed = 0
-  if !empty(s:current)
-    call system('git show '.s:current.' 2> '.(has('win32') ? 'nul' : '/dev/null'))
-    let managed = !v:shell_error
-  endif
-
-  if !managed
+  if empty(s:current)
     call clap#helper#echo_error('The current buffer is not in the working tree' . s:current)
```

Comment:
Return a List so that this error message can be shown in the display window.

```suggestion
    return ['The current buffer is not in the working tree' . s:current]
```